website/docs/quick-start-guide.md

```diff
@@ -216,9 +237,33 @@ can generate sample inserts and updates based on the the sample trip schema [her
 
 ## Create Table
 
+Before we go further, few terminologies to familiarize: 
+
+- **Table types**
```

Comment:
@nsivabalan   Since these are common background knowledge and nothing specific to Spark SQL, moving these concepts to even above the Setup step would be good. May be we can add a section called Background or Introduction. What do you think ?

Reply:
lets wait to hear from vinoth as well.

Reply:
We don't need anything like this here. lets just assume a table type and go from there. lets pick MoR vs CoW based what can show case breadth easily, same for partitioned vs non-partitioned.